api/src/functions/migrateBusinessesOnUserChange/app.ts

```diff
@@ -0,0 +1,35 @@
+import { createDynamoDbClient } from "@db/config/dynamoDbConfig";
+import { DynamoBusinessesDataClient } from "@db/DynamoBusinessesDataClient";
+import { DynamoUserDataClient } from "@db/DynamoUserDataClient";
+import {
+  BUSINESSES_TABLE,
+  DYNAMO_OFFLINE_PORT,
+  IS_DOCKER,
+  IS_OFFLINE,
+  STAGE,
+  USERS_TABLE,
+} from "@functions/config";
+import { LogWriter } from "@libs/logWriter";
+import { usersBusinessesMigration } from "@libs/usersBusinessesMigration";
+import { DynamoDBStreamEvent } from "aws-lambda";
+
+export default async function handler(event: DynamoDBStreamEvent): Promise<void> {
+  const logger = LogWriter(`MigrateBusinessesOnUserChange/${STAGE}`, "ApiLogs");
+
+  const dynamoDb = createDynamoDbClient(IS_OFFLINE, IS_DOCKER, DYNAMO_OFFLINE_PORT);
+  const usersDbClient = DynamoUserDataClient(dynamoDb, USERS_TABLE, logger);
+  const businessesDbClient = DynamoBusinessesDataClient(dynamoDb, BUSINESSES_TABLE);
+
+  try {
+    const isBusinessesDataMigrationEnabled =
+      process.env.ENABLE_BUSINESS_MIGRATION_FROM_USERS_TABLE === "true";
+    if (!isBusinessesDataMigrationEnabled) {
+      logger.LogInfo("Businesses migration is disabled");
```

Comment:
The current logs in this section are sufficient for identifying whether the migration flag is enabled or not. The logic here is straightforward: if the flag is enabled, we proceed to the migration logic, which already contains detailed logging for debugging. I don't believe additional logs are needed in this part of the code since the key conditions—whether the flag is enabled or not—are already captured. The migration logic itself is where the majority of the complexity lies, and that's where the relevant logs are in place to help debug any issues.